charts/pocket-id/Chart.yaml

```diff
@@ -5,7 +5,7 @@ description: |
   pocket-id is a simple and easy-to-use OIDC provider that allows users to authenticate
   with their passkeys to your services.
 home: https://pocket-id.org
-version: 1.3.1
+version: 1.4.0
```

Comment:
Please do not bump the version manually. I have an automation that will create a release and generate a changelong after this gets merged.